.github/actions/send-release-annotation-pixel/action.yml

```diff
@@ -0,0 +1,41 @@
+name: "Send Release Annotation Pixel"
+description: "Sends a pixel to annotate a new iOS or macOS client release"
+
+inputs:
+  version:
+    description: "The version number of the release"
+    required: true
+  platform:
+    description: "The platform of the release (ios or macos)"
+    required: true
+    type: choice
+    options:
+      - ios
+      - macos
+    default: "ios"
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Send Release Annotation Pixel"
+      shell: bash
+      run: |
+        if [[ "${{ inputs.platform }}" != "ios" && "${{ inputs.platform }}" != "macos" ]]; then
+          echo "Invalid platform: ${{ inputs.platform }}"
+          exit 1
+        fi
+
+        if [[ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]]; then
+          pixel_name="m_test_pixel_debug_${{ inputs.platform }}" # TODO rename
+        else
+          pixel_name="m_test_pixel_${{ inputs.platform }}" # TODO rename
+        fi
```

Comment:
I wanted to support firing the workflow directly via `workflow_dispatch` in case we need to ever test this, so the pixel has been given a different name in that case. Note that this is still using a debug name for both cases, I'll rename this once the PR is approved.